rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -179,6 +180,15 @@ public int getMaxDepth() {
         return 1024;
     }
 
+    public void markCallWithValuePerformed() {
+        callWithValuePerformed = true;
+//        result.markCallWithValuePerformed();
+    }
+
+    public boolean getCallWithValuePerformed() {
+        return callWithValuePerformed;
+    }
```

Comment:
this was useful on sergio's first iteration (which was using the `gasNeeded + SITPEND_CALL` upperbound), but now we don't need this field anymore, since we are estimating using the `maxGasUsed`. I'll leave a comment and remove it before merging it into master

Reply:
https://github.com/rsksmart/rskj/pull/1548/commits/96402e84a4e49487278d2f360aa1eba0aa5eafe2